engine/src/gfx/cockpit.cpp

```diff
@@ -679,8 +680,12 @@ float GameCockpit::LookupUnitStat(int stat, Unit *target) {
         case UnitImages<void>::CANDOCK_MODAL: {
             Unit *station = target->Target();
             if (station) {
-                if (station->CanDockWithMe(target, true) != -1) {
-                    if (station->CanDockWithMe(target, false) != -1) {
+                if(target->isUnit() != Vega_UnitType::planet ) {
+                    return (float) UnitImages<void>::NOMINAL;
+                }
+
+                if (CanDock(station, target, true) != -1) {
+                    if (CanDock(station, target, false) != -1) {
```

Comment:
This reads a bit strange, maybe a comment could help a casual reader

Reply:
Huh. The original code was:
             ```
   if (station->CanDockWithMe(target, true) != -1) {
                    if (station->CanDockWithMe(target, false) != -1) {
```
I've no idea why it this so.